Cleanse/CanonicalRepresentable.swift

```diff
@@ -57,12 +59,3 @@ extension Optional : CanonicalRepresentable {
     }
     
 #endif
-
```

Comment:
This a breaking change to the public API. What was the motivation behind removing it?

Reply:
`ImplicitlyUnwrappedOptional` has been removed as of Swift 4.1. This is necessary for the library to continue to build on that version and newer. Consumers will also have to migrate their code from using `ImplicitlyUnwrappedOptional` when adopting Swift 4.1 or greater.

Reply:
Silly me, this was commented out. Disregard.